ios/Classes/PMPlugin.m

```diff
@@ -119,13 +118,16 @@ -(UIViewController*) getCurrentViewController {
     return nil;
 }
 
--(void)presentLimited {
+-(void)presentLimited:(ResultHandler*) handler {
     if (@available(iOS 14, *)) {
         UIViewController* ctl = [self getCurrentViewController];
         if(!ctl){
+            [handler replyError:@"No valid controller"];
             return;
         }
-        [PHPhotoLibrary.sharedPhotoLibrary presentLimitedLibraryPickerFromViewController: ctl];
+        [PHPhotoLibrary.sharedPhotoLibrary presentLimitedLibraryPickerFromViewController: ctl completion:^{
```

Comment:
The api is only available in iOS 15.

Reply:
![image](https://user-images.githubusercontent.com/14145407/124224329-04327800-db38-11eb-8da5-67dd1f82bd36.png)